web/src/lib/components/asset-viewer/activity-viewer.svelte

```diff
@@ -51,9 +51,11 @@
   }
 
   $: {
-    if (previousAssetId != assetId) {
+    if (assetId && previousAssetId != assetId) {
       getReactions();
       previousAssetId = assetId;
+    } else if (assetId === undefined) {
+      getReactions();
```

Comment:
I think I would prefer
```typescript
if (assetId) {
    getReactions();
}
if (previousAssetId !== assetId) {
    previousAssetId = assetId;
}
```
This should work as well because `undefined !== null`.